packages/webex/package.json

```diff
@@ -37,17 +38,34 @@
       "envify"
     ]
   },
+  "scripts": {
```

Comment:
Why did we need these changes here ?

Reply:
@Kesari3008 we weren't building the webex package so far in the `next` branch. We were only building the remaining plugins and testing deployments to npm. As part of this change, we will be building the webex package as well. This was needed in order to get the calling bundle built as well.

Reply:
Let me clarify this a little bit. 
> This was needed in order to get the calling bundle built as well.

This change isn't needed to get the calling bundle built. It's required to build the `umd` folder under `packages/webex` which will be used to distribute to CDN.

Reply:
If we don't have these scripts, we observe the following error when doing `yarn run build` at webex-js-sdk level.
```
ERROR in ./packages/webex/src/calling.js 11:0-47
Module not found: Error: Can't resolve '@webex/calling' in '/Users/sreenara/Stuff/Repos/webex-js-sdk/packages/webex/src'
resolve '@webex/calling' in '/Users/sreenara/Stuff/Repos/webex-js-sdk/packages/webex/src'
  Parsed request is a module
```